packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/implicit/flow-diagram.md

```diff
@@ -0,0 +1,37 @@
+### Implicit
+
+![Implicit flow](/img/authorization/oauth-implicit-grant-flow.png "Sequence diagram that displays the back and forth between the resource owner, authorization server, and resource server for Implicit grant flow")
+
+<!-- Source for image. Generated using http://www.plantuml.com/plantuml/uml/
+
+skinparam monochrome true
+actor "Resource Owner (User)" as user
+participant "Client" as client
+participant "Authorization Server (Okta)" as okta
+participant "Resource Server (Your App)" as app
+
+autonumber "<b>#."
+client -> okta: Access token request to /authorize
+okta -> user: 302 redirect to authentication prompt
+user -> okta: Authentication & consent
+okta -> client: Access token response
+client -> app: Request with access token
+app -> client: Response
+
+-->
+The Implicit flow contains the following interaction steps:
+
+1. The Client sends a request to the Auth Server (Okta) for an access token.<br>
```

Comment:
do you need the <br> here? Two returns and a tab puts that content on a new line (with an empty line above) and indented under step 1. Does that not work?

Reply:
Yep. I'll implement this.